src/Ingridient.js

```diff
@@ -0,0 +1,9 @@
+class Ingridient {
+  constructor(id = 0, name = 'unknown', estimatedCostInCents = 0) {
+    this.id = id,
```

Comment:
Also, we just talked about fixing the spelling of ingredient. We'll do that before we merge!